drivers/flash/Kconfig.nor

```diff
@@ -9,6 +9,8 @@ menuconfig SPI_NOR
 	bool
 	prompt "SPI NOR Flash"
 	select FLASH_HAS_DRIVER_ENABLED
+	select FLASH_HAS_PAGE_LAYOUT
+	select FLASH_PAGE_LAYOUT
```

Comment:
these selects here are wrong and causing kconfig to go into a loop and breaks documentation.

Reply:
Have addressed this and shippable is passing now.